src/App.tsx

```diff
@@ -1,24 +1,40 @@
-import { createGlobalStyle } from "styled-components";
+import styled, { createGlobalStyle } from "styled-components";
+
+import ProjectDefinition from "./interfaces/Project";
+import { GET_PROJECT_BY_ID } from "./api";
+import { useFetch } from "./hooks/useFetch";
+import LoadingSpinner from "./components/LoadingSpinner";
 import Panel from "./components/Panel";
 import Project from "./components/Project";
-import getProject from "./repositories/getProject";
 
 const GlobalStyle = createGlobalStyle`
   font-family: Open Sans, Montserrat, Roboto
 `;
 
 export default function App(): JSX.Element {
-  const project = getProject(1);
+  const { data: project, isLoading } = useFetch<ProjectDefinition>(GET_PROJECT_BY_ID(1));
```

Comment:
This `GET_PROJECT_BY_ID` is imported from api constants folder. Lemme know if you have any preferences on how to handle APIs in FE!